docs/articles/documentation/test-api/test-code-structure.md

```diff
@@ -572,3 +573,163 @@ test('Fixture2Test3', () => {});
 
 // Only tests in Fixture1 and the Fixture2Test2 test are run
 ```
+
+## Inject Scripts into Tested Pages
+
+TestCafe allows you to inject custom scripts into pages visited during the tests. You can add scripts that mock standard API or provide helper functions.
+
+Use the `fixture.clientScripts` and `test.clientScripts` methods to add scripts to each page visited during a particular test or fixture.
+
+```text
+fixture.clientScripts( script[, script2[, ...[, scriptN]]] )
+```
+
+```text
+test.clientScripts( script[, script2[, ...[, scriptN]]] )
+```
+
+Parameter | Type     | Description
+--------- | -------- | ---------------------------------------------------------------------------
+`scripts` | String &#124; Object &#124; Array | Scripts to inject into the tested pages. Pass a [file path](#specify-the-path-to-a-javascript-file) or [code](#specify-the-script-code). You can also [specify pages](#provide-scripts-for-particular-pages) into which scripts should be injected.
+
+The `clientScripts` methods can take [multiple arguments and arrays](#pass-multiple-arguments).
+
+### Specify the Path to a JavaScript File
+
+Specify the JavaScript file path to inject the entire content of this file into the tested pages. You can pass a string or an object with the `path` property.
+
+```text
+fixture.clientScripts( path | { path } )
+```
+
+```text
+test.clientScripts( path | { path } )
+```
+
+Argument  | Type   | Description
+--------- | ------ | ---------------------------------------------------------------------------
+`path`    | String | The path to the JavaScript file whose content should be injected. Relative paths resolve from the directory that contains the current test file.
+
+**Example**
+
+```js
+fixture.clientScripts('assets/jquery.js');
+fixture.clientScripts({ path: 'assets/jquery.js' });
+```
+
+> You cannot combine the `path`, [module](#specify-the-module-name) and [content](#specify-the-script-code) properties.
+
+### Specify the Module Name
+
+Specify the module name to inject the module's content into the tested pages. Use a string or an object with the `module` property.
+
+```text
+fixture.clientScripts( module | { module } )
+```
+
+```text
+test.clientScripts( module | { module } )
+```
+
+Argument  | Type   | Description
+--------- | ------ | ---------------------------------------------------------------------------
+`module`  | String | The name of the module whose content should be injected. TestCafe uses [require.resolve](https://nodejs.org/api/modules.html#modules_require_resolve_request_options) to resolve the module location with Node.js machinery.
```

Comment:
Need to say that the module's entry point file will be injected:

`TestCafe will inject the content of the file which is the module's entry point. The module location will be searched with Node.js machinery.`